$NetBSD: patch-Makefile,v 1.1 2024/10/13 20:56:41 yhardy Exp $

Enable the OPL3 emulation for adlib music.
The MT32 driver needs munt, so disable it.

--- Makefile.orig	2024-10-12 22:21:37.310552587 +0000
+++ Makefile
@@ -19,8 +19,8 @@ SRCS = collision.cpp cpc_player.cpp cuts
 #CXXFLAGS += -DUSE_STATIC_SCALER
 #SCALERS  := scalers/scaler_nearest.cpp scalers/scaler_tv2x.cpp scalers/scaler_xbr.cpp
 
-#CXXFLAGS    += -DUSE_MIDI_DRIVER
-#MIDIDRIVERS := midi_driver_adlib.cpp midi_driver_mt32.cpp
+CXXFLAGS    += -DUSE_MIDI_DRIVER
+MIDIDRIVERS := midi_driver_adlib.cpp # midi_driver_mt32.cpp
 #MIDI_LIBS   := -lmt32emu
 
 LIBS = $(MIDI_LIBS) $(MODPLUG_LIBS) $(SDL_LIBS) $(TREMOR_LIBS) $(ZLIB_LIBS)
